<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HW1374" name="Test-chrome-Mobile" tests="1" failures="1" timestamp="18 Jan 2018 10:41:09 GMT" time="8.705" errors="0">
  <testcase name="Login" time="0.024" classname="com.web.automation.scripts.LoginValidations">
    <failure type="java.lang.AssertionError" message="Test Case Failed because - Failed to complete Login.">
      <![CDATA[java.lang.AssertionError: Test Case Failed because - Failed to complete Login.
	at com.web.automation.scripts.LoginValidations.TestCaseStatus(LoginValidations.java:33)
	at com.web.automation.scripts.LoginValidations.Login(LoginValidations.java:85)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
</testsuite> <!-- Test-chrome-Mobile -->
